### PR TITLE
fix OCP-25762 OCP-25931

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/common_ui_elements.xyaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ select_resource_item:
 click_on_resource_name:
   element:
     selector:
-      xpath: //a[contains(@class,'co-resource-item__resource-name') and text()='<item>']
+      xpath: //span[contains(@class,'co-resource-item__resource-name') and text()='<item>']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/common_ui_elements.xyaml]: Maybe `//*[contains(@class,'co-resource-item__resource-name') and text()='<item>']` ? It looks it changes very often

**Comment by SSshahan** [comment]
@liangxia  updated base the comments. please help to merge , thx

**Pull request merged**